docs/guides/generic_recognizers.md

```diff
@@ -387,11 +385,16 @@ Some of these algorithms have extra parameters, some do not, but all do have the
 
 ### [PostProcessing](xref:AudioAnalysisTools.Events.Types.EventPostProcessing.PostProcessingConfig)
 
-Post-processing of events is performed after event detection. However it is important to understand that post-processing is performed once for each of the DecibelThresholds. As an example: suppose you have three decibel thresholds (6, 9 and 12 dB is a typical set of values) in each of two profiles. All the events detected at threshold 6 dB (by both profiles) will be collected together and subjected to the post processing steps. Typically some or all of the events may fail to be accepted as "true" events based on your post-processing parameters. Then all the events detected at 9 dB will be collected and independently subjected to post-processing. Then, likewise, all events detected at the 12 dB threshold will be post-processed. In other words, one round of post-processing is performed for each decibel threshold. This sequence of multiple post-processing steps gives rise to one or more temporally nested events. Think of them as Russion doll events! The final post-processing step is to remove all but the longest duration event in any nested set of events.
+Post-processing of events is performed after event detection. Post-processing is performed once for each of the DecibelThresholds. As an example: suppose you have three decibel thresholds (6, 9 and 12 dB is a typical set of values) in each of two profiles. There will be three rounds of post-processing:
+- All the events detected at threshold 6 dB (by both profiles) will be collected together and subjected to the post-processing steps. Typically some or all of the events may fail to be accepted as "true" events based on your post-processing parameters.
+- Next all the events detected at 9 dB will be collected and independently subjected to post-processing.
+- Next all events detected at the 12 dB threshold will be post-processed. 
+
+This sequence of multiple post-processing steps gives rise to one or more temporally nested events. Think of them as Russion doll events! The final post-processing step is to remove all but the longest duration event in any nested set of events.
```

Comment:
```suggestion
Post-processing of events is performed after event detection. Post-processing is performed once for each of the DecibelThresholds. 
As an example: suppose you have three decibel thresholds (6, 9 and 12 dB is a typical set of values) in each of two profiles.
There will be three rounds of post-processing:

- All the events detected at threshold 6 dB (by both profiles) will be collected together and subjected to the 
  post-processing steps. Typically some or all of the events may fail to be accepted as "true" events based on your
  post-processing parameters.
- Next all the events detected at 9 dB will be collected and independently subjected to post-processing.
- Next all events detected at the 12 dB threshold will be post-processed. 

This sequence of multiple post-processing steps gives rise to one or more temporally nested events. Think of them as Russion doll events!
The final post-processing step is to remove all but the longest duration event in any nested set of events.
```

Reply:
I have rewritten parts of this but not clear what the issue was/is.